### ISSUE TITLE
Doubled colon problem - Unknown command ::f

### ISSUE THREAD

**Issue by khud** [khud/sparklin#10]
To print function-valued variable we should use `::f` instead `f` the same as in the `:type` command. This cause error message:
```
kotlin> fun f() = 1
kotlin> :t ::f
kotlin.reflect.KFunction0<kotlin.Int>
kotlin> ::f
Unknown command ::f
```

**Comment by khud** [comment]
If we will fix it we would have something like this 
```
kotlin> fun f() = 1
kotlin> ::f
res1: kotlin.reflect.KFunction0<kotlin.Int> = fun Line_3.f(): kotlin.Int
```
Is it really okey or `Line_3.f()` should be replaced to `f()`?

**Comment by khud** [comment]
Fixed in 0.1.2, but result will be in form `fun Line_3.f(): kotlin.Int`.
